### DISCUSSION TITLE
I still haven't solved this problem

### DISCUSSION THREAD

**Discussion by 123xyl** [openmopac/mopac#116] [Q&A]
![image](https://user-images.githubusercontent.com/96686347/190351245-7b306933-41da-4d83-843c-2bb69b090d9a.png)

**Comment by godotalgorithm** [comment]
As I told you in #107, this problem should be fixed in the development version right now. You are still building in a directory named `mopac-22.0.3`, which indicates that you did not download an updated version of the MOPAC source code that fixes your problem. You can download the development version from [here](https://github.com/openmopac/mopac/archive/refs/heads/main.zip), which now uses `command_argument_count` instead of `iargc` when the flang compiler is detected.

**Reply by 123xyl** [reply]
thank you
